keentools_facebuilder/interface/updater.py

```diff
@@ -133,3 +165,71 @@ def execute(self, context):
         uc.skip_update(res.plugin_name, res.version)
         FBUpdater.clear_message()
         return {'FINISHED'}
+
+
+MIN_TIME_BETWEEN_REMINDERS = 86400  # 24 hours in seconds
```

Comment:
_MIN_TIME_BETWEEN_REMINDERS (just add underline at the start to mark it as internal variable)